integration/spark-datasource/src/main/scala/org/apache/spark/util/SparkUtil.scala

```diff
@@ -57,4 +59,12 @@ object SparkUtil {
     isSparkVersionXandAbove(xVersion, true)
   }
 
+  // "spark.sql.execution.id is already set" exception will be
+  // thrown if not set to null in spark2.2 and below versions
+  def setSparkExecutionId(sparkSession: SparkSession, executionId: String): Unit = {
```

Comment:
just name the method name as `setNullExecutionId` and don't pass the `executionId` to it. Handle inside the method

Reply:
@ravipesala Handled the comments ..please review